portfolio/src/main/webapp/index.html

```diff
@@ -7,7 +7,7 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
+    <div id="content" onload="getData()">
```

Comment:
You don't need to add this function on your index page since you aren't adding the comments here anywhere